src/bootstrap.php

```diff
@@ -2224,7 +2224,7 @@ function (Accept $type, string $id) use ($app) {
         $headers
     );
 })->before($app['negotiate.accept'](
-    'application/vnd.elife.search+json; version=2',
+//    'application/vnd.elife.search+json; version=2', TODO: somehow doesn't allow the tests pass! Need to discuss.
```

Comment:
This won't pass until we upgrade the api-raml to include the new version. The media type needs to be declared in the schema otherwise it will error on the validate step.